management-interface.go

```diff
@@ -289,6 +293,301 @@ func NetworkHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "network.html", state)
 }
 
+//getNetworkSsid -gets the ssid of the network associated with this id
+func getNetworkSsid(networkID string) (string, error) {
+	var out []byte
+	err := error(nil)
+	//get network
+	out, err = exec.Command("wpa_cli", "get_network", networkID, "ssid").Output()
+	if err != nil {
+		log.Printf("error executing wpa_cli get_network %s - error %s output %s", networkID, err.Error(), out)
+		return "", err
+	}
+
+	stdOut := string(out)
+	scanner := bufio.NewScanner(strings.NewReader(stdOut))
+	scanner.Scan() //skip 1st line interface line
+	scanner.Scan()
+	ssid := scanner.Text()
+	return ssid, err
+
+}
+
+//deleteNetwork - delets network with id using wpa_cli and saves config
+func deleteNetwork(id string) error {
+	var out []byte
+	err := error(nil)
+
+	//check if is bushnet
+	var ssid string
+	ssid, err = getNetworkSsid(id)
```

Comment:
in Go, the convention is to write acronyms in upper case unless it's the first word in the identifier. So in this case `getNetworkSSID`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms